getting-started/project-setup.adoc

```diff
@@ -26,60 +26,26 @@ xref:../partials/rules-sample-pom.adoc[Sample Rules/Decision Application POM Tem
 
 . Open the project in VS Code.
-  From here we can see the workspace contents, and if we expand the contents of `/src/main` you can see the creation of several artifacts. 
-  Within `java` you have a `GreetingResource.java` and within `resources` you have an `application.properties` and `pricing.dmn` file. 
-  These are sample files that can be later modified or deleted.
-+
-image::workspace-layout.png[VSCode Workspace Layout]
-
+  Within the explorer are the basic artifacts created by maven:
+.. `src/main/java`
+.. `src/main/java/com/mycompany/app/App.java`
+.. `test/java/com/mycompany/app/AppTest.java`
+.. and others.
 . Install {VS_CODE_BUNDLE_LINK}[*{VS_CODE_BUNDLE}*].
-  This gives you access to the visual editors for DMN and BPMN files.
-  It also allows you to create and edit testing scenarios for projects.
+This gives you access to the visual editors for DMN and BPMN files.
+It also allows you to create and edit testing scenarios for projects.
 +
 image::bamoe-vscode-marketplace.png[{VS_CODE_BUNDLE} within the marketplace]
 
-. Click the `pricing.dmn` file to open it. 
-  When you do so, you may be greeted with a message similar to `The current DMN does not have any Diagram associated with it. Do you want to auto-generate it?` 
-  If so click `Auto-generate Diagram` to automap the DMN locations.
-+
-image::automatic-layout.png[Automatic Layout]
-
-. When the diagram opens you will see something similar to below, and you can start exploring it. 
-  The DMN consists of two inputs: *Age* and *Previous incidents?*, which are used to make the decision, *Base price*.
-+
-image::dmn-first-view.png[DMN First view]
+== Next Steps
 
-. If you click the *Age* input data node and then click the *Properties* icon on the right, you will open a pane for the input.
-+
-image::properties-open.png[DMN Properties]
-
-. Within this pane, you can see information about the input *Age*, namely that it is a number and its input name. 
-  Other changes can be made to the object, including the color of the node, font size, etc.
-+
-image::properties-expanded.png[DMN Properties Expanded]
-
-. To view the Decision, click the square decision node and select the `Edit` button to enter the decision for *Base Price*.
-+
-image::open-decision.png[Edit DMN Decision]
-
-. From here you will see the Decision Table that is associated with the Base Price decision. 
-  From here you will see two (*2*) input columns (`Age` and `Previous Incidents`), as well as one output column (`Base price`) all with their types below them. 
-  These types are controlled from the properties panel similarly to how they were opened when we previously looked at `Age`. 
-  This decision has 4 different rows that could fire, with a Hit Policy of `UNIQUE` signified by the **U** in the top left corner of the table. 
-  A decision writer could make any comments they want to the table and have them saved with the decision here.
-+
-image::dt-stable.png[Viewing DMN Decision]
```

Comment:
I think we are loosing content here

Reply:
I think this is the same comment as above